src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -8,10 +8,6 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
   <ItemGroup>
-    <Compile Include="HttpWebRequestTest.cs" />
-    <Compile Include="HttpWebResponseTest.cs" />
-    <Compile Include="RequestStreamTest.cs" />
-    <Compile Include="WebRequestTest.cs" />
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
```

Comment:
Please remove this change.  In general, you should keep your changes small and specific to the bug you are wishing to fix.